### PR TITLE
Handle null object for negate of datetime/date/time

### PR DESCRIPTION
**Pull request by nguhoa**
Fix issue https://github.com/microsoft/Power-Fx/issues/1294

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs [hunk 1]
```diff
@@ -474,11 +474,16 @@ internal static partial class Library
 #endregion
 
 #region Unary Operator Implementations
-        private static NumberValue NumericNegate(IRContext irContext, NumberValue[] args)
-        {
+        private static FormulaValue NumericNegate(IRContext irContext, NumberValue[] args)
+        {
+            if (args.Length == 0 || args[0] == null)
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs:479]: >if (args.Length == 0 || args[0] == null) [](http://example.com/codeflow?start=12&length=40)

How does this happen?

Comment (nguhoa) [reply src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs:479]: it does not. i saw it pass null value in to negate func.

Comment (MikeStall) [reply src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs:479]: That may be the bug ... do you know where those nulls are coming from?

Comment (CarlosFigueiraMSFT) [reply src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs:479]: This is just masking the symptom of the problem, not addressing the root cause. It should be valid to negate a time/date/datetime value. Something like `-dateValue` should be equivalent to `NumberToDate(Negate(DateToNumber(dateValue)))` - and similar for time/datetime.

**Review by CarlosFigueiraMSFT — CHANGES_REQUESTED** [review]
:clock1:

**Pull request closed without merging**